### PR TITLE
bundle: add autogenerated bundle files

### PR DESCRIPTION
**Pull request by mmirecki**
The bundle files can be generated using "make bundle" with
appropriate parameters provided

To generate the files in this commit, the following parameters
were used with the command:
VERSION=4.10.0 IMG=quay.io/fromani/rte-operator:latest make bundle

Note that when running the command for the first time or when the
bundle files are removed, the command will ask additional questions,
asking for:
- operator name
- operator description
- provider name
- provider url
- maintainers
- operator keywords

### CODE REVIEW

File: bundle/manifests/numaresources-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+                  protocol: TCP
+                resources: {}
+              - args:
+                - --platform=kubernetes
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                command:
+                - /bin/numaresources-operator
+                image: quay.io/fromani/rte-operator:latest
```
Comment (fromanirh) [inline bundle/manifests/numaresources-operator.clusterserviceversion.yaml:201]: this needs to be updated

File: bundle/manifests/numaresources-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+                  name: https
+                  protocol: TCP
+                resources: {}
+              - args:
+                - --platform=kubernetes
+                - --health-probe-bind-address=:8081
+                - --metrics-bind-address=127.0.0.1:8080
+                - --leader-elect
+                command:
+                - /bin/numaresources-operator
```
Comment (fromanirh) [inline bundle/manifests/numaresources-operator.clusterserviceversion.yaml:200]: don't we miss the parameters here?

**Review by fromanirh — COMMENTED** [review]
thanks for the updates. It seems to me we need some minor tweaks to the operator command line/image and we are good to go.

File: bundle/manifests/numaresources-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (mmirecki) [reply bundle/manifests/numaresources-operator.clusterserviceversion.yaml:201]: quay.io/fromani/numaresources-operator:v4.10.0 ?

Comment (fromanirh) [reply bundle/manifests/numaresources-operator.clusterserviceversion.yaml:201]: `quay.io/openshift-kni/numaresources-operator:v4.10.0`

Comment (fromanirh) [reply bundle/manifests/numaresources-operator.clusterserviceversion.yaml:201]: > `quay.io/openshift-kni/numaresources-operator:v4.10.0`

correcting myself, looking at the tags prow generates (https://quay.io/repository/openshift-kni/performance-addon-operator?tab=tags) it should be

`quay.io/openshift-kni/numaresources-operator:4.10-snapshot`

**Comment by mmirecki** [comment]
Replaced by #4

**Pull request closed without merging**